versions/0_2/flake.nix

```diff
@@ -7,7 +7,7 @@
       };
 
       lair = {
-        url = "github:holochain/lair/lair_keystore-v0.2.4";
+        url = "github:holochain/lair/lair_keystore-v0.3.0";
```

Comment:
This can't go in yet? Needs Holochain 0.2.2

Reply:
got it. favorably this would be caught by an integration test that spins up lair and a conductor and does some rudimentary self-checks.

Reply:
https://github.com/holochain/holochain/pull/2748 introduces a test that i'd expect this PR to fail at